app/src/main/res/values/strings.xml

```diff
@@ -172,4 +172,6 @@
     <string name="cd_navdrawer_user_picture">User Profile Image</string>
     <string name="title_signing_needed">Sign-in Needed</string>
     <string name="signin">Sign-in</string>
+    <string name="feedback_message_required">Message required</string>
+    <string name="feedback_invalid_email">Input valid email address</string>

```

Comment:
You are a native speaker @Splaktar 
Please help us with the sentences in the app.